api/pkg/apis/v1alpha1/model/instance.go

```diff
@@ -31,7 +31,6 @@ type (
 		Target       TargetSelector    `json:"target,omitempty"`
 		Topologies   []TopologySpec    `json:"topologies,omitempty"`
 		Pipelines    []PipelineSpec    `json:"pipelines,omitempty"`
-		Generation   string            `json:"generation,omitempty"`
 		Version      string            `json:"version,omitempty"`
```

Comment:
Can we also look at how this "Version" is used? This was introduced by AIO, not sure what its functionality is.

Reply:
This seems to be introduced by @LeiQL for all versioned objects (version + rootResource property). Can you please confirm it?

Reply:
Correct. Being remove in PR #323.